solr/contrib/jaegertracer-configurator/src/java/org/apache/solr/jaeger/JaegerTracerConfigurator.java

```diff
@@ -31,7 +31,5 @@ public Tracer getTracer() {
   }
 
   @Override
-  public void init(@SuppressWarnings({"rawtypes"})NamedList args) {
-  }
-
+  public void init(@SuppressWarnings({"rawtypes"}) NamedList args) {}
```

Comment:
This conflict with one of the changes I made, it's easy to resolve though, just delete everything!

Reply:
> This conflict with one of the changes I made, it's easy to resolve though, just delete everything!

Good catch! I assume #169 are the other changes. Would you like to proceed with that pull request first or would you prefer this PR here be merged first? Either approach would work for me.

Reply:
No, it was #165, once you resolve the conflict here then this should be good to merge!